beta/src/pages/apis/render.md

```diff
@@ -4,16 +4,74 @@ title: render()
 
 <Intro>
 
-`render` renders a piece of [JSX](/learn/writing-markup-with-jsx) ("React element") into a browser DOM container node. It instructs React to change the DOM inside of the `container` so that it matches the passed JSX.
+`render` renders a piece of [JSX](/learn/writing-markup-with-jsx) ("React node") into a browser DOM container node.
+
+</Intro>
+
+## On this page {/*on-this-page*/}
+
+- [Reference](#reference)
+  - [`render(reactNode, container)`](#render)
+  - [`render(reactNode, container, callback)`](#render-callback)
+- [Usage](#usage)
+  - [Rendering the root component](#rendering-the-root-component)
+  - [Rendering multiple roots](#rendering-multiple-roots)
+  - [Updating the rendered tree](#updating-the-rendered-tree)
+
+## Reference {/*reference*/}
+
+### `render(reactNode, container)` {/*render*/}
+
+Call `render` to display a React component inside a `container` browser DOM element.
 
 ```js
+const container = document.getElementById('root');
 render(<App />, container);
-render(<App />, container, callback);
 ```
 
-</Intro>
+React will take over managing the DOM inside of the `container`.
+
+An app fully built with React will usually only have one `render` call with its root component.  A page that uses "sprinkles" of React for parts of the page may have as many `render` calls as needed. [See examples below.](#usage)
+
+#### Parameters {/*parameters*/}
+
+* `reactNode`: A *React node* that you want to display. This will usually be a piece of JSX like `<App />`, but you can also pass a React elmenent constructed with [`createElement()`](/TODO), a string, a number, `null`, or `undefined`. 
```

Comment:
```suggestion
* `reactNode`: A *React node* that you want to display. This will usually be a piece of JSX like `<App />`, but you can also pass a React element constructed with [`createElement()`](/TODO), a string, a number, `null`, or `undefined`. 
```

Reply:
Not sure how this got missed 😶‍🌫️